in_toto/in_toto_keygen.py

```diff
@@ -99,11 +104,17 @@ def main():
 
   try:
     if args.prompt:
-      in_toto.util.prompt_generate_and_write_rsa_keypair(args.name, args.bits)
+      if args.type == in_toto.util.KeyType.RSA:
+        in_toto.util.prompt_generate_and_write_rsa_keypair(args.name,
+                                                           args.bits)
+      elif args.type == in_toto.util.KeyType.ED25519:
+        in_toto.util.prompt_generate_and_write_ed25519_keypair(args.name)
       sys.exit(0)
     else:
-      in_toto.util.generate_and_write_rsa_keypair(args.name, args.bits,
-        password=None)
+      if args.type == in_toto.util.KeyType.RSA:
+        in_toto.util.generate_and_write_rsa_keypair(args.name)
+      elif args.type == in_toto.util.KeyType.ED25519:
+        in_toto.util.generate_and_write_ed25519_keypair(args.name)
```

Comment:
This dangling `elif` causes coverage to think that there is a case which I haven't tested. How should I tackle it?

Reply:
There are few more lines of code that are not hit by coverage.
https://travis-ci.org/in-toto/in-toto/jobs/383287249

I recommend you only use `#pragma: no cover` if you know for sure that Travis cannot execute a specific line.  In this case, shouldn't line 116 be covered by a unit test?